source/js/buyers-guide/components/filter/filter.jsx

```diff
@@ -82,6 +84,14 @@ export default class Filter extends React.Component {
 
       this.updateWithCSSvalues(update);
       this.setState(update);
+
+      if(DNT.allowTracking) {
```

Comment:
in `bg-main.js`

```js
if (DNT.allowTracking) {
  ReactGA.initialize(`UA-87658599-6`);
  ReactGA.pageview(window.location.pathname);
}
```

ReactGA won't get initiated if DNT.allowTracking is turned off. I wonder if we still need to check for that value here?

Reply:
Ah, it won't get _initialized to our ga-id_ but it'll still be an active object, so its better to not even trigger it at all if we know that tracking is not appreciated. It's probably even better to make a little wrapper lib that proxies ReactGA calls, but a little more work than should happen in this PR specifically (so I filed https://github.com/mozilla/foundation.mozilla.org/issues/2193 for that work)

Reply:
Cool. Thanks for filing that ticket!